config/sway/config.d/50-greybeard.conf

```diff
@@ -7,8 +7,13 @@
 bindsym $mod+Shift+Return exec $$term -e distrobox enter
 
 # set sway-launcher-desktop as launcher
-for_window [app_id="^launcher$"] floating enable, sticky enable, resize set 30 ppt 30 ppt, border pixel 10
-set $menu exec $term -a launcher -e sway-launcher-desktop
+# for_window [app_id="^launcher$"] floating enable, sticky enable, resize set 30 ppt 30 ppt, border pixel 10
+# set $menu exec $term -a launcher -e sway-launcher-desktop
+
+### Set rofi as launcher, and enable hotkeys for rofi-network-manager and rofi-bluetooth
+set $menu exec rofi -combi-modi window,drun,ssh,combi -terminal foot -theme arthur -font "hack 10" -show combi
+bindsym $mod+n exec rofi-network-manager
+bindsym $mod+b exec rofi-bluetooth
```

Comment:
There is also `https://git.cepl.eu/cgit/nmcli-rofi/tree/README.md`, which could get included (I may package it, if required). I don’t know what you use for when you are around coffee-house/airport Wi-Fi hotspots.